main.js

```diff
@@ -516,7 +516,9 @@ exports.main = function (options, callbacks) {
     let win = viewFor(window);
     let mainWindow = win.document.getElementById('main-window');
     let tabbrowser = win.document.getElementById('tabbrowser-tabs');
-    if (mainWindow.getAttribute('doNotReverse') !== 'true' && options.loadReason === 'upgrade' && prefs.prefs.opentabstop === true) {
+    // opentabstop pref
+    // eslint-disable-next-line no-constant-condition
+    if (mainWindow.getAttribute('doNotReverse') !== 'true' && options.loadReason === 'upgrade' && false) {
```

Comment:
We could just change this all to false, but I'm happy doing it this way.

Reply:
I think this way is easier should we ever decide to restore it